app/src/main/res/values/strings.xml

```diff
@@ -88,6 +88,9 @@
     <string name="card_screen_add_review">Add A Review</string>
     <string name="card_screen_report">Report</string>
     <string name="card_screen_no_image">No images for this parking yet</string>
+    <string name="card_screen_confirm_upload">Do you want to add this image to the parking spot ? </string>
+    <string name="card_screen__accept_upload">Yes</string>
+    <string name="card_screen__cancel_upload">No</string>
```

Comment:
We already have general resources for Yes and No, no need to define them again for this specific purpose